### DISCUSSION TITLE
Icons for Half-Life and it's expansions?

### DISCUSSION THREAD

**Discussion by fourTH3MA573R** [elrumo/macOS_Big_Sur_icons_replacements#1402] [Icon request] [answered]
I would be really grateful if someone could make these in big Sur's style! 🙌

(Half-Life)
https://preview.redd.it/dktdwaydgms61.png?width=1890&format=png&auto=webp&s=55be5dc9d5f5a14ba0d52087b7b98902bfd5f203

(Half-Life Opposing Force)
https://preview.redd.it/v4b4izeegms61.png?width=1890&format=png&auto=webp&s=0e9633c34bbb773d0f7aedfa6e570180464ebbb9

(half-Life Blue Shift)
https://preview.redd.it/4eh1h55dgms61.png?width=1890&format=png&auto=webp&s=452a332f807b4490cfaf7dcdc0921617b0a42067

**Comment by rahman09abd** [comment, marked as answer]
I've uploaded (status: waiting) and you can check my profile later [mmnk09](https://macosicons.com/#/u/mmnk09)

**Reply by fourTH3MA573R** [reply]
Thank you! These look pretty good, but I've already made a few on my own (thanks image2icon.)  I still appreciate the effort though.
